app/js/components/manageApps/Login.jsx

```diff
@@ -9,14 +10,36 @@ class Login extends Component {
       username: '',
       password: '',
       show: false
-    }
+    };
+
     this.handleLogin = this.handleLogin.bind(this);
     this.handleShow = this.handleShow.bind(this);
     this.handleClose = this.handleClose.bind(this);
   }
+
+  componentWillMount(){
+    this.checkLoginStatus();
+  }
+
+  checkLoginStatus(){
+    this.fetchLocation('/v1/session').then((response) => {
+      (response.authenticated) ? hashHistory.push('/') : null;
+    });
+  }
+
+  fetchLocation(url) {
+    const apiHelper = new ApiHelper(null);
```

Comment:
Is the value null being used specifically? Why not call the ApiHelper constructor without arguments instead

Reply:
Fixed.